packages/terra-date-time-picker/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Added
+  * Added keyboard shortcuts `N`, `-`, `+`, `A`, and `P` for auto-setting date and time values.
```

Comment:
Do we want to call out `T` also?

Reply:
So I wasn't sure how accurate that would be, because since we pull in date-picker, as soon as date-picker consumed its changes `T` was supported? so these changes mostly deal with the "extra" that has to happen with time shortcuts and dates.

Reply:
From a consumer or end-users POV, they will not see or care about the internal distinction between this component and others it consumes. As a simple solution: how about adding a link in the Date Time Picker doc to reference the additional time commands?

Reply:
Oh sorry, I see you indeed have the `T` referenced in the doc!! My bad 🤦🏼